### PR TITLE
Add missing gem and add migrate task

### PR DESCRIPTION
**Pull request by jvanbaarsen**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -4,6 +4,7 @@ gem "pg", "~> 0.18"
 gem "puma", "~> 3.0"
 gem "sass-rails", "~> 5.0"
 gem "coffee-rails", "~> 4.2"
+gem "uglifier", "~> 3.0.2"
```
Comment (michiels) [inline Gemfile:7]: Interesting. Why is this needed?

Comment (jvanbaarsen) [reply Gemfile:7]: I don't know, the push was failing for not having that gem in here (It's a default rails gem which I removed).

Comment (michiels) [reply Gemfile:7]: Ah ok, probably the asset compile task needs it.

Comment (jvanbaarsen) [reply Gemfile:7]: Yeah I think so. Probably to minify the css and JS

**Comment by jvanbaarsen** [comment]
@michiels :shipit: ?

**Comment by michiels** [comment]
@jvanbaarsen yeah of course, minor change :)
